### ISSUE TITLE
Issue: Player adding a nation causes invited nation to create an error in mysql.

### ISSUE THREAD

**Issue by Phyther** [TownyAdvanced/Towny#2926]
This ticket template is for bug/error reports. If you are suggesting a feature and not reporting a bug please delete all the text here and begin your ticket Title with Suggestion:

Note: In checking this issue, I switched my version back to 0.92.0.0

### What steps will reproduce the problem?
1. Player does /n add [nation_name]

### What is the expected output?
1. Error is issued to inviting player

### What is your Towny version?
0.92.0.11

### What is your spigot/craftbukkit version?
paper 1.12.2 (git-paper-1391)

### Please use Pastebin.com to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly. If you do skip this step then your ticket will most likely be deleted and you will be asked to resubmit.**
1. Your full server startup from the logs\latest.log : https://pastebin.com/utDi1Ffh
2. Your Towny config.yml (if using MySQL please remove password,) : https://pastebin.com/0hLn2E2Q
3. Your townyperms.yml : https://pastebin.com/ZJTtY03X
4. Your log's error : https://pastebin.com/iDEWQxZK

(optional) If this is to do with permissions, 
5. Your permissions file :

(optional) If this is to do with chat,
6. Your chatconfig.yml :
7. Your channels.yml :

**Comment by Articdive** [comment]
Uhm, he tried inviting out_Heaven and not outer_heaven
Thought it shouldn't be giving us a nullpointer there

**Comment by Phyther** [comment]
Yeah, good point, but after that he did do a /n add Outer_heaven, no error following it, and outer heaven crashed the server 20 minutes later with a null error

**Comment by Articdive** [comment]
Can be assigned to me

**Comment by Articdive** [comment]
I have contact with @Phyther via discord, still waiting on a response regarding if the issue fixed itself while not using MariaDB

**Comment by Phyther** [comment]
So for whatever reason, I can't get OracleMySQL to work on my home machine nor the server machine. I'm sorry I'm delayed with responding to this.
